tests/api/test_metrics.py

```diff
@@ -23,12 +23,90 @@
 )
 
 
-# we need to mock time.monotonic because on windows it does not
-# capture enough precision for these tests to not be flaky
-@pytest.fixture
-def mock_time_monotonic():
-    with mock.patch("time.monotonic", side_effect=count()) as mocked_time_monotonic:
-        yield mocked_time_monotonic
+def test_metrics_no_counters():
```

Comment:
All counter tests moved from `test_metrics_counters.py` to consolidate the files and keep to the convention